src/main/java/io/appium/java_client/android/HasSettings.java

```diff
@@ -76,8 +76,7 @@ default void ignoreUnimportantViews(Boolean compress) {
      * @param timeout in milliseconds, 0 would reset to its default value
```

Comment:
negative value would reset, not  0, because in some circumstances, 0 would be used to disable timeouts in settings on purpose.  for all configurator settings. see [here](https://github.com/truebit/appium/blob/master/docs/en/advanced-concepts/settings.md#android-uiautomator-configurator)